src/.nuget/NuGetRestore.ps1

```diff
@@ -1,5 +1,6 @@
 $NuGetExe = "$PSScriptRoot\NuGet.exe"
 
+& $NuGetExe install xunit.runner.console -version 2.0.0 -OutputDirectory "$PSScriptRoot\..\..\packages"
```

Comment:
@tannergooding The unit test assemblies in dotnet/roslyn-analyzers target xUnit 2.0.0, not 1.9.2. Tests are definitely being found, run, and reported.